integration-tests/gradle/projects/it-android-0/gradle/wrapper/gradle-wrapper.properties

```
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-bin.zip
```

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-6.9.2-bin.zip
```

Bump all these `6.9` to `6.9.2`?